src/settings.js

```diff
@@ -1943,12 +1943,12 @@ var SPLIT_MODULE = 0;
 // How to calculate reverse dependencies (dependencies from JS functions to
 // native functions) prior to linking native code with wasm-ld.  This option
 // has three possible values:
-// 'auto': (default) inspect the object code passed to the linker (by running
-//         llvm-nm) all all inputs and use the map in deps_info.py to
-//         determines the set of additional dependencies.
+// 'auto': (default) Inspect the object code passed to the linker (by running
+//         llvm-nm) and use the map in deps_info.py to determine the set of
```

Comment:
I think I meant to write: `(by running llvm-nm on all input)`

Reply:
Done